### DISCUSSION TITLE
Transcoding, and album-level fields

### DISCUSSION THREAD

**Discussion by bearcatsandor** [quodlibet/quodlibet#4486] [Q&A] [answered]
Ive been using QuodLibet since version 0.4, and setting aside the fact that the feeling of losing my mind is a common occurrence for me, I would _swear_ that I'd seen a transcode dialog in it.  The idea was that you could copy files to the root of your music folder structure and it would transcode, then move/rename the files for you. There was even an option for Wavpack which is what I use.  I've looked through all the  plugins and preferences and can't find it. I can't even find the file-naming rules. Was it a plugin?

Secondly, is there a way to pull album-level tags from Musicbrainz? With beets tagger I have access to the catalog-number, the media (cd, digital, sacd etc) and i can add custom album level fields that tell me about the remastering (great to separate out my MoFi and DCC discs). I'd love to be able to filter in the album browsers with that.

Thank you,

Bearcat

**Comment by declension** [comment, marked as answer]
> I would swear that I'd seen a transcode dialog in it. The idea was that you could copy files to the root of your music folder structure and it would transcode, then move/rename the files for you. There was even an option for Wavpack which is what I use. 

I don't think I've ever seen that, but it could be a _long_ time ago I guess.


> I can't even find the file-naming rules. Was it a plugin?

[Renaming files](https://quodlibet.readthedocs.io/en/latest/guide/renaming_files.html) maybe? 


> Secondly, is there a way to pull album-level tags from Musicbrainz?

I guess it depends what tags, but the Musicbrainz plugin pulls all the main track ones, plus album title, albumartist, MB Release ID and various other things.

**Reply by bearcatsandor** [reply]
Thank you. I found the file-naming rules under right-click --> Edit Tags. It doesn't quite handle 'the' the way I'd like it to, but a regex should fix that (maybe). I'd like my sorts to be "Bob Dylan" and "Beatles, the"

I also found that I can add "label" and "catalognumber" to the Cover Grid.  I'm still a little confused by the database. I can add "label" in the cover grid so that tells me that field is at the album level, however there's a "labelid" at in the track browser that is empty.
